docs_src/microservices/application/services/AppRecordReplay/GettingStarted.md

```diff
@@ -0,0 +1,210 @@
+
+!!! warning
+    Actual replay rates are not exact and will vary depending on OS load.
+
+Optionally set the `repeatCount` which determines how many times to replay the recorded session. Defaults to once if not set or set to 0.
+
+The source device services should be stopped from producing any new events prior to starting the replay session. In a real use case where the devices are no longer available, the device service(s) would not have any actual devices connected to generate events. In this guide we simple stop the **Device Virtual** container, which can be done from Portainer or with a docker CLI command.
+
+!!! example - "Example stopping Device Virtual container"
+    ```
+    docker stop edgex-device-virtual
+    ```
+
+After above is completed simply press `Send` in PostMan to start the replay session .
+
+#### Check Replay Status
+
+The status of a replay session can be checked while it is running or after is has completed. Review the **Replay Status** GET API in the [API Reference.md](ApiReference) section and use the **Replay Status** request from the PostMan collection referenced above. Press `Send` in PostMan to get the replay status.
+
+!!! note
+    This API will always return a **202 - Accepted** response. If there were issues with the replay, the message field will contain the reason.
+
+!!! example - "Example Replay Status responses"
+    ```json
+    Replay currently running response:
+    {
+        "running": true,
+        "eventCount": 5,
+        "duration": 2773407921,
+        "repeatCount": 0,
+        "Message": ""
+    }
+    
+    ```
+
+    Replay completed response:
+    {
+        "running": false,
+        "eventCount": 20,
+        "duration": 20015604050,
+        "repeatCount": 2,
+        "Message": ""
+    }
+    
+    Replay canceled response::
+    {
+        "running": false,
+        "eventCount": 2,
+        "duration": 0,
+        "repeatCount": 0,
+        "Message": "replay canceled"
+    }
+    
+    No Replay response:
+    {
+        "running": false,
+        "eventCount": 0,
+        "duration": 0,
+        "repeatCount": 0,
+        "Message": "no replay running or previously run"
+    }
+    ```
+
+#### Cancel Replay
+
+A replay session can be canceled while it is running. Review the **Cancel Replay** DELETE API in the [API Reference](ApiReference.md) section and use the **Cancel Replay** request from the PostMan collection referenced above. Press `Send` in PostMan to cancel the replay session.
+
+!!! note
+    This API will return a **202 - Accepted** response if the replay can be canceled, otherwise it will return an error such as "***failed to cancel replay: no replay currently running***"
+
+### Export a Recorded Session
+
+The current recorded session can be exported so that data can be saved to the file system. Review the **Export** GET API in the [API Reference](ApiReference.md) section and use the **Export Recording** requests from the PostMan collection referenced above.
```

Comment:
```suggestion
The current recorded session can be exported so that data can be saved to the file system. Review the **Export** GET API in the [API Reference](ApiReference.md) section and use the **Export Recording** requests from the Postman collection referenced above.
```

Reply:
fixed